### PR TITLE
Create portfolio2

### PR DESCRIPTION
**Pull request by Wolfmikerson**
This pull request includes:

1- Added Work Section
2- Added About MySelf Section
3- Used CSS Grid and FlexBox combined

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Wolfmikerson,** 

### **Highlight**

- The PR is descriptive.
- The commit msg are clear.
- Linters are passing

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

- I don't see the changes you did add like ( HTML/CSS), I only see the README.md file.
- Please add some changes so you can get the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -90,7 +90,7 @@ <h2 class="support-text-2 default-custom-fonts lets-text-font">LETS CONNECT</h2>
           <li><img class="counter" src="images/Counter.svg" alt="Counter"></li>
```
Comment (DanteAlonsoHT) [inline index.html:90]: - [ ] Fix margin/padding according to the Figma design you selected.
- [ ] Fix padding for "Get my resume" `button` tag.
- [ ] [OPTIONAL] Fix HTML image in the "Languages" section.

Figma Design | Yours

image

Comment (DanteAlonsoHT) [inline index.html:90]: - [ ] Fix margin/padding in this section.
- [ ] Also you can add the right background-color for the project technologies list "HTML", "css", "javascript".

Figma Design | Yours

image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @Wolfmikerson, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] You License link in your README file is broken, please fix it.
- [ ] Your link to go to your LinkedIn account doesn't look professional, look:
Yours
```
- LinkedIn: [LinkedIn] https://www.linkedin.com/in/wolf-m-jacques-27a38612a/)
```
It should be:
```
- LinkedIn: [Wolf_LinkedIn](https://www.linkedin.com/in/wolf-m-jacques-27a38612a/)
```

- [ ] Add the links for your social media in the "Authors" section.

- [ ]  _Check the comments under the review._

## Optional suggestions

- [ ]  _Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

<hr>

### I know you have the capabilities enough to finish these changes required in time 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 2]
```diff
@@ -7,16 +7,16 @@ Portfolio: setup a mobile version
...
 ## Authors
 
 👤 **Wolf M. Jacques**
 
-- GitHub: Wolfmikerson
-- Twitter: @wolfmikerson
-- LinkedIn: [LinkedIn] https://www.linkedin.com/in/wolf-m-jacques-27a38612a/)
+- GitHub: Wolfmikerson (https://github.com/Wolfmikerson)
+- Twitter: @wolfmikerson (https://twitter.com/MikersonWolf)
+- LinkedIn: [Wolf_LinkedIn](https://www.linkedin.com/in/wolf-m-jacques-27a38612a/)
```
Comment (DanteAlonsoHT) [inline README.md:20]: - [ ]  [OPTIONAL] For Github and Twitter, kindly do the same structure as your LinkedIn profile.

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @Wolfmikerson, 

### Your project is complete! , Please add the OPTIONAL changes required (check in the comments below) 😉  After that, feel free to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**